test/karma.conf.js

```diff
@@ -15,15 +15,17 @@ module.exports = function(config) {
     // list of files / patterns to load in the browser
     files: [
       '<%= resolve_asset("vis.js") %>',
-      '<%= resolve_asset('jquery.js') %>',
+      '<%= resolve_asset("jquery.js") %>',
       '<%= resolve_asset("angular.js") %>',
       '<%= resolve_asset("angular-mocks.js") %>',
       '<%= resolve_asset("underscore.js") %>',
+      '<%= resolve_asset("moment.js") %>',
       'test/angular/test_helper.js',
       'app/assets/javascripts/app.js',
-      'app/assets/javascripts/controllers/**/*.js',
-      'app/assets/javascripts/directives/**/*.js',
+      'app/assets/javascripts/filters/**/*.js',
       'app/assets/javascripts/services/**/*.js',
+      'app/assets/javascripts/directives/**/*.js',
```

Comment:
Actually, this change was unnecessary, I was trying to debug some problem and thought the order mattered. Turned out it didn't, but I forgot about this change. I'll change it back for aesthetics! :)